### ISSUE TITLE
`src` pushed to npm (`react-cosmos-test`)

### ISSUE THREAD

**Issue by victormoukhortov** [react-cosmos/react-cosmos#611]
### What's up?

It seems that the ES6 `src` files have been pushed to npm. This messes up module resolution in jest, where `require('enzyme')` in `lib/enzyme.js` resolves to `src/enzyme.js` instead of `node_modules/enzyme/...`

### Mkay, tell me more...

Delete `src` folder from npm registry for package `react-cosmos-test`

**Comment by skidding** [comment]
Hmm, that's bad. We have to check this for every Cosmos package. I was under the impression that only what's under [`package.files`](https://github.com/react-cosmos/react-cosmos/blob/56077795420e70a5f314a3fa737918fc8cb7e152/packages/react-cosmos-test/package.json#L7-L110 will be published to npm.

@victormoukhortov thanks for reporting!

**Comment by gir** [comment]
As a workaround, for now, you can just add this to your package.json
```
{
  ...
  "scripts": {
    ...
    "postinstall": "rm -rf ./node_modules/react-cosmos-test/src",
    ...
  },
  ...
}
```

**Comment by skidding** [comment]
A few leads on this one:
- [Lerna doesn't seem to be the issue](https://github.com/lerna/lerna/issues/1282#issuecomment-373046797)
- [Yarn seems to have a hard time respecting package.json:files](https://github.com/yarnpkg/yarn/issues/2822#issuecomment-374505604)

Will keep you posted as soon as I learn anything new, and any outside help is appreciated.

**Comment by jbgraug** [comment]
I tried both "npmClient": "yarn" and "npmClient": "npm" and both behave the same way.
I have local libraries swhich get symlinked into the main project so the postinstall script is not a solution for me (i don't want to delete my src files. ;)
We should be able to configure, somehow, the folder to be symlinked (files attribute in package.json)

**Comment by skidding** [comment]
Fixed this in #637. `src` dirs should no longer be present in `4.0.0-beta.11` and above. But of course you're getting the beta release with this version.
